LICENSE-2.0.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Can we rename this file to `LICENSE` so that it gets picked up correctly by Github as a proper license file?

You end up with the Repo being correctly identified as Apache-2.0 this way.

e.g. 

https://github.com/snowplow/snowplow-android-tracker/blob/master/LICENSE
vs 
https://github.com/snowplow/snowplow-ruby-tracker/blob/master/LICENSE-2.0.txt